src/runtime/dune

```diff
@@ -1,4 +1,5 @@
 (library
  (name ppx_version_runtime)
  (public_name ppx_version.runtime)
- (libraries core_kernel))
+ (libraries core_kernel ppxlib sexplib yojson)
```

Comment:
the reason the [checks](https://github.com/o1-labs/ppx_version/pull/33/checks) are not passing is that `yojson` needs to be installed here: https://github.com/o1-labs/ppx_version/blob/master/.github/workflows/changes.yml#L20

and for `opam pin` to work it needs to be added to the opam package file here: https://github.com/o1-labs/ppx_version/blob/master/ppx_version.opam#L14 (important otherwise we can't publish these changes to our opam repository)

Reply:
(and same for other dependencies)

Reply:
Thanks, builds now.